File: Python/for Loops.py
# for loops = a statement that will execute it's block of code
#             a limited amount of times
#
#             while loop = unlimited
#             for loop = limited

import time

# i means index

for i in range(10):
    print(i+1)

for i in range(50,100+1,2):
    print(i)

for i in "Peter Martens":
    print(i)

for seconds in range(10,0,-1):
    print(seconds)
    time.sleep(1)
print("Happy New Year!")

File: C#/BroCode/ForLoops.cs
using System;

namespace MyFirstProgram
{
    class Program
    {
        static void Main(string[] args)
        {

            // for loop = repeats some code a FINITE amount of times

            // Count up to 10
            for (int i = 1; i <= 10; i++)
            {
                Console.WriteLine(i);
            }

            // Count down from 10
            for (int i = 10; i > 0; i--)
            {
                Console.WriteLine(i);
            }
            Console.WriteLine("HAPPY NEW YEAR!");

            Console.ReadKey();
        }
    }
}
